.Rbuildignore

```diff
@@ -23,3 +23,4 @@
 ^man/figures/usethis-og-1280x640\.png$
 ^internal$
 ^revdep-cloud$
+^CODE_OF_CONDUCT\.md$
```

Comment:
Doesn't this show up below `.github/` and is therefore already `.Rbuildignore`d?